lib/wrapper.js

```diff
@@ -57,14 +59,15 @@ class RequestWrapper {
   }
 
   _handleResponseError(error) {
-    logger.error('fatal_error', error.message, this._getLogParameters());
+    logger.fromError('fatal_error', error, this._getLogParameters());
     throw new SuiteRequestError(error.message, 500);
   }
 
   _handleResponse(response) {
     if (response.statusCode >= 400) {
-      logger.error('server_error', response.body.replyText, this._getLogParameters({
-        code: response.statusCode
+      logger.error('server_error',this._getLogParameters({
```

Comment:
same here, function name in the first parameter or more context would help with debugging